modules/accredited_representative_portal/db/migrate/20250313190720_create_ar_icn_temporary_identifiers.rb

```diff
@@ -0,0 +1,9 @@
+class CreateArIcnTemporaryIdentifiers < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ar_icn_temporary_identifiers, id: :uuid do |t|
```

Comment:
I was trying to think through this name and I think I like that it says temporary as a friendly synonym for ephemeral and drives home the point that these shouldn't really be treated as stable / permanent identifiers.

Is it accurate that we don't want think of these as stable? For example, there could be a link for some sub-resource of a claimant that could become invalid due to the impermanent nature of the claimant ID, but the app can still regenerate a way to get back to that resource?